TOOLS.md

```
@@ -28,6 +28,7 @@ This document lists all available Claude Code tools, Cursor tools, and Gemini Ge
 - [Aipcc Adr Assistant](#aipcc-adr-assistant-gemini-gems)
 - [Technical Spike & Investigation](#technical-spike-&-investigation-gemini-gems)
 - [Email Copilot](#email-copilot-gemini-gems)
+- [Tl;Dev](#tl;dev-gemini-gems)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)